crypto/bn/bn_lib.c

```diff
@@ -132,13 +132,57 @@ int BN_num_bits_word(BN_ULONG l)
     return bits;
 }
 
+/*
+ * This function still leaks `a->dmax`: it's caller's responsibility to
+ * expand the input `a` in advance to a public length.
+ */
+static ossl_inline
+int BN_num_bits_consttime(const BIGNUM *a)
```

Comment:
Since this is an internal function only, should it instead be called `bn_num_bits_consttime`?

Reply:
Done!